scripts/package_antares_web.sh

```diff
@@ -31,6 +31,7 @@ echo "INFO: Preparing the Git Commit ID..."
 git log -1 HEAD --format=%H > ${RESOURCES_DIR}/commit_id
 
 echo "INFO: Generating the Desktop version of the Web Application..."
+rm -rf ${DIST_DIR}
```

Comment:
Why do you remove this ? Because you use this path later in the script, ex : 
```pyinstaller --distpath ${DIST_DIR} AntaresWebWin.spec```

Reply:
This directory removal is required only in scenarios where the script is executed twice. This situation occurs in development mode.